pkgs/applications/misc/lswt/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromSourcehut, nixos, wayland }:
+
+stdenv.mkDerivation rec {
+  pname = "lswt";
+  version = "1.0.4";
+
+  buildInputs = [ wayland ];
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Orwa7sV56AeznEcq/Xj5qj4PALMxq0CI+ZnXuY4JYE0=";
+  };
+
```

Comment:
```suggestion

  buildInputs = [ wayland ];

```

Reply:
did this get reverted? I see the wayland input, but this line got dropped

Reply:
Good catch. I was trying to apply via the GH web UI that change got skipped. Fixed.